dao-contracts/tests/features/variables/voting_start_after_job_submission.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Voting start after job submission
+  VotingStartAfterJobSubmission variable tests
+
+  Background:
+    Given following balances
+      | account          | CSPR balance | REP balance  | REP stake  | is_kyced | is_va |
+      | JobPoster        | 1000         | 0            | 0          | true     | false |
+      | InternalWorker   | 0            | 1000         | 0          | true     | true  |
+      | VA1              | 0            | 1000         | 0          | true     | true  |
+    And following configuration
+      | key                           | value        |
+    # | VotingStartAfterJobSubmission | 2 days       |
```

Comment:
Could you left just one entry here? `2 days` should work, no?

Reply:
It's just easier to get what "2 days" stands for rather than the same value in seconds. Moreover, a few lines below we write "2 days passed", IMO it makes the test more readable.